ansible_deployer/command_line.py

```diff
@@ -367,7 +367,8 @@ def get_playbooks(config: dict, options: dict):
     for play in play_names:
         for item in config["tasks"]["play_items"]:
             if item["name"] == play:
-                if item["skip"]:
+                skip = item.get("skip", [])
```

Comment:
Does that fix something? if/for sequence above is just different

Reply:
Moved to #50 
It changes handling of field "skip" in tasks config. If there is no such field (we agreed on it being optional) empty list will be assigned and following condition will not be checked.